src/main/java/elara/command/DeleteCommand.java

```diff
@@ -0,0 +1,30 @@
+package elara.command;
+
+import elara.storage.Storage;
+import elara.task.InvalidInputException;
+import elara.task.TaskList;
+import elara.ui.Ui;
+
+public class DeleteCommand implements Command {
+    private final String fullInput;
+
+    public DeleteCommand(String taskDetails) {
+        this.fullInput = taskDetails;
+    }
+
+    @Override
+    public void execute(TaskList taskList, Ui ui, Storage storage) throws InvalidInputException {
```

Comment:
Would it be better to add a JavaDoc above this method?

Reply:
have added in later commits